index.html

```diff
@@ -1,10 +1,47 @@
 <!DOCTYPE html>
-<html lang>
+<html lang="en-US" class="dew-solar dew-wet dew-tab dew-hyper dew-shift dew-diy" id="mix">
+
 <meta charset="utf-8">
-<title>...</title>
 <meta name="viewport" content="width=device-width, initial-scale=1">
-<meta name="theme-color" content="seashell">
-<meta name="description" content="...">
-<link rel="stylesheet" href="wall.css">
-<link rel="help" href="https://github.com/webmural/wall">
+
+<title>I CREATE&copy; web sample with I above EA</title>
+
+<meta name="theme-color" content="#404">
+<meta name="color-scheme" content="dark light">
+
+<style media=(color) data-help="https://s9a.page/dew">
+.dew-shift,.dew-shift *{transition:var(--dew-shift)}.dew-wet,.dew-wet *{background:var(--dew-oil);color:var(--dew-ink)}.dew-rim,.dew-rim *{border-color:var(--dew-rim)}.dew-hyper,.dew-hyper :any-link{filter:var(--dew-hyper)}.dew-tab :focus,.dew-tab:focus{filter:var(--dew-focus);outline-color:var(--dew-locus)}.dew-field:disabled{filter:var(--dew-vapor)}.dew-lunar.dew-lunar,.dew-media.dew-media,.dew-polar.dew-polar,.dew-solar.dew-solar{fill:var(--dew-oil);background:var(--dew-oil);color:var(--dew-ink)}:root{--dew-hyper:none;--dew-media:var(--dew-solar);--dew-polar:var(--dew-lunar);--dew-oil:var(--dew-media);--dew-ink:var(--dew-polar);--dew-rim:var(--dew-ink);--dew-shift:none;--dew-solar:mistyrose;--dew-lunar:darkviolet;--dew-locus:currentColor;--dew-focus:invert(100%);--dew-hover:none;--dew-vapor:opacity(60%)}:any-link:hover{--dew-hyper:var(--dew-hover)}:any-link:focus{--dew-hyper:var(--dew-focus)}.dew-lunar,.dew-polar{--dew-oil:var(--dew-polar);--dew-ink:var(--dew-media)}@media (prefers-color-scheme:dark){:root{--dew-media:var(--dew-lunar);--dew-polar:var(--dew-solar)}}
+</style>
```

Comment:
`<style>` like this is an ideal way to load [dew](https://github.com/s9a/dew#dew) because it's fast local and prevents <b>f</b>lash <b>o</b>f <b>u</b>nstyled <b>c</b>olor